contrib/rootless-cni-infra/Containerfile

```diff
@@ -33,3 +33,5 @@ COPY --from=dnsname /dnsname /opt/cni/bin
 COPY rootless-cni-infra /usr/local/bin
 ENV CNI_PATH=/opt/cni/bin
 CMD ["sleep", "infinity"]
+
+ENV ROOTLESS_CNI_INFRA_VERSION=1
```

Comment:
If you plan to build this in the near future using a build trigger on quay.io, this will need to be named `Dockerfile`.  If not, then no worries.

Reply:
Thanks for the hint! I don't plan to auto-build the image as we have to manage the tags. Whenever we change something in this folder and build a new image, we have to bump this version number and the tag.